lib/train/transports/local.rb

```diff
@@ -36,6 +48,95 @@ def local?
         true
       end
 
+      def run_powershell_using_named_pipe(pipe_location, script)
+        pipe = nil
+        # Try to acquire pipe for 10 seconds with 0.1 second intervals.
+        # Removing this can result in instability due to the pipe being
+        # temporarily unavailable.
+        100.times do
+          begin
+            pipe = open(pipe_location, 'r+')
```

Comment:
Do we need to close the pipe?

Reply:
Interesting idea...after consulting with @jquick I think we should be able to do that. I'll get to work!